### PR TITLE
Add nullalbe exchange on company search search

### PR DESCRIPTION
**Pull request by walterfranssen**

### CODE REVIEW

File: Tests/CompanyValuation/CompanyValuationTests.cs [hunk 1]
```diff
@@ -363,6 +363,19 @@ public async Task SearchByTickerAsync()
             Assert.Equal("AGS.BR", firstResult.Symbol);
         }
 
+        [Fact]
+        public async Task SearchByTickerWithoutExchangeAsync()
+        {
+            var result = await api.SearchByTickerAsync("AGS", null, 5);
```
Comment (Matthiee) [inline Tests/CompanyValuation/CompanyValuationTests.cs:369]:
```suggestion
            var result = await api.SearchByTickerAsync("AGS", limit: 5);
```

**Review by Matthiee — CHANGES_REQUESTED** [review]
Thank you for the PR, it looks good!
However some small changes are needed before it can be merged.

**Comment by walterfranssen** [comment]
Do you know why 128 test are failing?

**Comment by Matthiee** [comment]
@walterfranssen There is only 1 unit test failing see my suggestion here: https://github.com/MatthiWare/FinancialModelingPrep.NET/pull/35/files#r894752437

After that it looks like it can be merged!

**Comment by walterfranssen** [comment]
I change the symbol too AAPL, is in my subscription. Works on my machine :)

**Review by Matthiee — APPROVED** [review]

**Comment by Matthiee** [comment]
@walterfranssen thank you for your contribution!

**Pull request merged**